src/utils/formatCurrencyAmount.ts

```diff
@@ -1,41 +1,17 @@
-import { Currency, CurrencyAmount, Fraction, Price } from '@uniswap/sdk-core'
-import { DEFAULT_LOCALE, SupportedLocale } from 'constants/locales'
-import JSBI from 'jsbi'
-import formatLocaleNumber from 'utils/formatLocaleNumber'
+import { Currency, CurrencyAmount, Price } from '@uniswap/sdk-core'
 
-export function formatCurrencyAmount(
-  amount: CurrencyAmount<Currency> | undefined,
-  sigFigs: number,
-  locale: SupportedLocale = DEFAULT_LOCALE,
-  fixedDecimals?: number
-): string {
-  if (!amount) {
-    return '-'
-  }
+import {
+  currencyAmountToPreciseFloat,
+  formatDollar,
+  formatTransactionAmount,
+  priceToPreciseFloat,
+} from './formatNumbers'
 
-  if (JSBI.equal(amount.quotient, JSBI.BigInt(0))) {
-    return '0'
-  }
-
-  if (amount.divide(amount.decimalScale).lessThan(new Fraction(1, 100000))) {
-    return `<${formatLocaleNumber({ number: 0.00001, locale, sigFigs, fixedDecimals })}`
-  }
-
-  return formatLocaleNumber({ number: amount, locale, sigFigs, fixedDecimals })
+export function formatCurrencyAmount(amount: CurrencyAmount<Currency> | undefined, isUsdPrice = false): string {
```

Comment:
wonder if we should take a config object as the second param, so it's clearer in code. `formatCurrencyAmount(currency, { isUsdPrice: true })`

Reply:
If you do not, you should annotate what the second parameter is in all usages:

    formatCurrencyAmount(usdcAmount, /* isUsdPrice= */ true)

Reply:
done

Reply:
is this done? https://github.com/Uniswap/widgets/pull/254/files#diff-4813987ddbdd14c353441866798a8843ba0fbb546503f4a7b134b995ab8be885R68